bokehjs/src/lib/models/glyphs/index.ts

```diff
@@ -6,32 +6,34 @@ export {Block}           from "./block"
 export {Circle}          from "./circle"
 export {Ellipse}         from "./ellipse"
 export {Glyph}           from "./glyph"
-export {HArea}           from "./harea"
 export {HAreaStep}       from "./harea_step"
+export {HArea}           from "./harea"
```

Comment:
I don't understand these re-orderings
```
In [4]: "HArea" < "HAreaStep"
Out[4]: True
```

Reply:
I use vim's (or equivalent) `:sort` command, which is non-semantic, thus it compares `S` < `}`.

Reply:
I resorted this the way eslint would. At some point, when there are fewer open PRs, I will most likely enable relevant lint rules to systematically organize imports and exports.